feature-libs/cart/wish-list/core/store/effects/wish-list.effect.ts

```diff
@@ -160,51 +154,45 @@ export class WishListEffects {
   );
 
   @Effect()
-  setWishListId$: Observable<CartActions.SetCartTypeIndex | undefined> =
-    this.actions$.pipe(
-      ofType(
-        WishListActions.CREATE_WISH_LIST_SUCCESS,
-        WishListActions.LOAD_WISH_LIST_SUCCESS,
-        CartActions.CLEAR_CART_STATE
-      ),
-      map((action: Action) => {
-        switch (action.type) {
-          case WishListActions.CREATE_WISH_LIST_SUCCESS:
-          case WishListActions.LOAD_WISH_LIST_SUCCESS: {
-            return new CartActions.SetCartTypeIndex({
-              cartType: CartType.WISH_LIST,
-              cartId: (action as StateUtils.EntitySuccessAction).meta
-                .entityId as string,
-            });
-          }
-          case CartActions.CLEAR_CART_STATE:
-            return new CartActions.SetCartTypeIndex({
-              cartType: CartType.WISH_LIST,
-              cartId: '',
-            });
```

Comment:
Why don't we need to clear the wiish list id in the cart type index anymore?

Reply:
`CLEAR_CART_STATE` is done on reducer directly. When action is `CLEAR_CART_STATE`, the whole `index` will be reset.